content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,156 @@ aliases:
 ---
 
 Use the `influxd upgrade` command to upgrade InfluxDB 1.x to InfluxDB 2.0.
-This command copies all data stored in 1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and
+The `upgrade` command copies all data stored in 1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and
```

Comment:
This feels really light...  meaning, the upgrade command exists to allow you to perform an in-place upgrade from InfluxDB 1.8.x to InfluxDB 2.0.x.  

Here's an overview of what the upgrade command does -->

Reply:
5 things...

Reply:
Adding this material now. 👍